app/pages/project/floating-ips/FloatingIpsPage.tsx

```diff
@@ -184,17 +184,18 @@ export function FloatingIpsPage() {
 
   return (
     <>
-      <PageHeader className="!mb-12">
+      <PageHeader>
         <PageTitle icon={<IpGlobal24Icon />}>Floating IPs</PageTitle>
+        <DocsPopover
+          heading="Floating IPs"
+          icon={<IpGlobal16Icon />}
+          summary="Floating IPs can be assigned to target instances, making it easier to host services from a consistent address."
```

Comment:
This is the right direction — I like the gesture at the purpose — but it's kind of vague somehow. This isn't the right wording but here is what I would want to convey.

> A floating IP is an IP address with an independent existence that can be attached and detached from instances.

"Independent existence" feels kind of ridiculous; "persistent" is more in the right direction but has a technical meaning that is misleading. Maybe independent existence is tolerable. Claude suggests other wordings that are the same words but feel better:

> Floating IPs exist independently of instances, allowing them to be dynamically attached and detached as needed.

Reply:
Will go with that for now, though happy to revise if we come up with improvements.